attestation/src/extensions/pkix/authkeyid.rs

```diff
@@ -10,7 +10,7 @@ use const_oid::{AssociatedOid, ObjectIdentifier};
 use der::asn1::{OctetStringRef, UIntRef};
 use der::Sequence;
 
-pub(crate) const AUTH_KEY_ID_EXTENSION_LEN: usize = 32;
+pub(crate) const AUTH_KEY_ID_EXTENSION_LEN: usize = 64;
```

Comment:
The commit message uses hex values with the 0x prefix.

Reply:
No, the commit message uses decimal values. 32 is not enough for 20 bytes long ID that translate to 40 bytes when encoding them to a readable string.